tests/drivers/spi/spi_error_cases/boards/nrf54h20dk_nrf54h20_cpuppr.conf

```diff
@@ -0,0 +1 @@
+CONFIG_ASSERT=n
```

Comment:
Why? Is it too heavy with assertions enabled? A brief comment would be helpful.

Reply:
added a comment